plugins/org.jkiss.dbeaver.ui.editors.hex/src/org/jkiss/dbeaver/ui/editors/binary/pref/HexPreferencesManager.java

```diff
@@ -317,18 +318,7 @@ private void populateFixedCharWidthFontsAsync()
         FontData fontData = getNextFontData();
         if (!fontsRejected.contains(fontData.getName())) {
```

Comment:
What about filtering-out Windows-specific fonts that start with `@`? They're vertically aligned, so they can't be used as a monospace font in this situation.

https://devblogs.microsoft.com/oldnewthing/20120719-00/?p=7093